### DISCUSSION TITLE
UI problem

### DISCUSSION THREAD

**Discussion by Gulshan45** [telekom-security/tpotce#1395] [Q&A]
i have installed t-pot on my laptop with debian 11 via post install method. I have checked the t-pot services it shows active but BLACKHOLE is disabled. After every few minutes t-pot becomes inactive and i have to restart the services. Some of the services are also down like iphoney, kibana, logstash, map_data. I'm also not able to get the UI on browser using IP:64294/64295,64297. How can i solve this problem of UI?

**Comment by Gulshan45** [comment]
![WhatsApp Image 2023-08-24 at 10 17 34](https://github.com/telekom-security/tpotce/assets/55939218/e9b0d65b-3a4e-4f71-b11d-872eb6863e63)

**Reply by t3chn0m4g3** [reply]
Probably interfering services like cupsd.

**Reply by Gulshan45** [reply]
How should I resolve it?
I can see the map_web web page but it just shows the map without any data. I have tried to install it on a server having 250gb Ram and 2 tb HDD but still the problem remains same. I can't access the kibana and majority of time it shows down status.
